TRANSLATING.md

```diff
@@ -133,3 +133,48 @@ pnpm run add-language
 The CLI will prompt you for a tag and name for the new language as described above. Follow the instructions and the wizard will create a basic set of files to get started translating that language.
 
 Update the placeholder content in the newly created files, commit them, and away you go!
+
+### What to translate: Frontmatter
+
+Our pages are generated from Markdown files which have frontmatter properties. These are variables that hold information about the page (values) that we later use to specify the page's title, description, and other special data. 
+
+Here's an example file showing the **properties** of `layout`, `title`, `description` and `i18nReady` along with their corresponding values for this page.
+
+```
+---
+layout: ~/layouts/MainLayout.astro
+title: Data Fetching
+description: Learn how to fetch remote data with Astro using the fetch API.
+i18nReady: true
+---
+
+// Rest of the file's content is here...
+```
+**tl/dr: Translate only some values, never translate properties!**
+
+The frontmatter **properties** themselves, like `title` and `description` should not be translated, as doing so would cause a runtime error and break our CI.
+
+The frontmatter **values** that should be translated are `title` and `description`. 
```

Comment:
```suggestion
The only frontmatter **values** that should be translated are those corresponding to the `title` and `layout` properties: "*Data Fetching*" and "*Learn how to fetch remote data with Astro using the fetch API*." 
```
I realized this could be confusing - don't translate title and description! Only translate title and description! lol

Reply:
Agree! I'll go with your idea. Just a thing, you mistook `description` with `layout`.
```suggestion
The only frontmatter **values** that should be translated are those corresponding to the `title` and `description` properties: "*Data Fetching*" and "*Learn how to fetch remote data with Astro using the fetch API*."
```